examples/authentication.test.js

```diff
@@ -36,10 +37,10 @@ describe('Some tests for authentication for our api', () => {
       aud: 'private',
       iss: 'master',
     })
-    const { status } = await request
+    const { status } = await supertest(app)
       .get('/')
       .set('Authorization', `Bearer ${access_token}`)
-    expect(status).toEqual(401)
+    expect(status).toEqual(500)
```

Comment:
The express middleware behaves a bit different when it cannot find the key. I mean to a certain degree this really is a server error and not an authentication error, so I do not care about the specific code as long as access is denied.